src/patternfly/components/FormControl/form-control.scss

```diff
@@ -241,6 +245,17 @@ $pf-c-form-control--m-clock--Coordinates: "M256 8C119 8 8 119 8 256s111 248 248
         --pf-c-form-control--focus--BorderBottomColor: var(--pf-c-form-control--readonly--focus--BorderBottomColor);
       }
     }
+
+    &.pf-m-plain {
+      --pf-c-form-control--readonly--BackgroundColor: var(--pf-c-form-control--readonly--m-plain--BackgroundColor);
+      --pf-c-form-control--inset--base: var(--pf-c-form-control--readonly--m-plain--inset--base);
+
+      border-color: transparent;
+
+      &:focus {
+        --pf-c-form-control--focus--BorderBottomColor: transparent;
+      }
```

Comment:
```suggestion
```

Doesn't look like you need this...
1. The specificity of `.pf-c-form-control[readonly]:not(.pf-m-success):not([aria-invalid=true]):focus` (whew 😅) overrides this var declaration, but that doesn't really matter since...
2. Where you declared [`border-color: transparent;`](https://github.com/patternfly/patternfly/blob/1a8ce3dc99f3554c3069d299afc56f186d8bfb63/src/patternfly/components/FormControl/form-control.scss#L253), you're no longer referencing the custom property/var for the `.pf-m-plain` border-color, and since the [original `:focus` selector](https://github.com/patternfly/patternfly/blob/1a8ce3dc99f3554c3069d299afc56f186d8bfb63/src/patternfly/components/FormControl/form-control.scss#L266) to update the border color updates a var, your declaration of the `border-color` property will also apply to `:focus`.